src/Centreon/Domain/Monitoring/Resource.php

```diff
@@ -56,6 +55,11 @@ class Resource
     public const TYPE_SERVICE = 'service';
     public const TYPE_HOST = 'host';
 
+    /**
+     * @var string|null
```

Comment:
A uuid cannot be null because it represents an existing system resource.
Therefore the uuid should be part of the constructor.